packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -784,7 +789,7 @@ export class ContainerRuntime extends EventEmitter
                 }
                 const snapshotTree = value as ISnapshotTree;
                 // Need to rip through snapshot.
-                const { pkg, snapshotFormatVersion } = readAndParseFromBlobs<IFluidDataStoreAttributes>(
+                const { pkg, snapshotFormatVersion, isRootStore } = readAndParseFromBlobs<IFluidDataStoreAttributes>(
```

Comment:
Consider this PR as a prototype, but once you are ready to commit things, we may want to go with multiple smaller PRs.
One addressing differentiation of root vs. non-root data stores would be nice to have in separate PR.

Reply:
Yep, agreed. I will need to break it down. I wanted to get an idea of how people feel about this prototype and if things look good, I will break it down into smaller pieces and check them in separately.